headers/Samples/FSSample.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+#include <vector>
+
+class FSSample
+{
+private:
+    const size_t SIZE;
+    double* data;
```

Comment:
The goal of FSSample is to manage a STATICALLY-instantiated array under the hood